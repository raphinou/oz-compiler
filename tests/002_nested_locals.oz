% The top local declared and initialises variables A and B.
% The nested local redeclares B and initialises it with another value.
% In the nested local, we then show both A and B.
% In the outer local, we show A and B to check it has the top level value.
local
   A=5
   B=6
in
   local
      B=7
   in
      {Show A}
      {Show B}
   end
   {Show A}
   {Show B}
end
