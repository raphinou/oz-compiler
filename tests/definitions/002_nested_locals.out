5
7
5
6
